HyperPlay-Gaming/hyperplay-desktop-client
# [CI] Add mac arm64 runner

Build arm64 mac on a m1 runner to fix bytecode error

This uses a manual trigger to save $. Once it is merged main you will be able to manually trigger for any branch here https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/actions/workflows/build-mac-arm64.yml


### Testing
1. Verify the error from 0.7.1 on m1
2. Download `mac-dmg-arm64` https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/actions/runs/6424604877?pr=574
3. Sign the app with `xattr -r -d com.apple.quarantine /Applications/HyperPlay.app`
4. Verify the app launches without error

---

Use the following Checklist if you have changed something on the Backend or Frontend:
